CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## [1.49](https://github.com/brave/brave-ios/milestone/106)
+
+Web3
+
+- Added Solana Swap integration into existing swap UI using Jupiter as the DEX aggregator ([#6847](https://github.com/brave/brave-ios/issues/6847))
+- Added message for Submitted/Completed state change after a transaction is submitted. ([#6960](https://github.com/brave/brave-ios/issues/6960))
+- Added UI to display ‘signed’ status for Ethereum transactions ([#6606](https://github.com/brave/brave-ios/issues/6606))
+- Added support for sending to ENS wallet address ([#5787](https://github.com/brave/brave-ios/issues/5787) )
+- Added auto-complete when entering token address for custom Solana assets  ([#6990](https://github.com/brave/brave-ios/issues/6990))
+- Fixed wallet issue where the Original Password field was shown when adding Solana Account ([#7175](https://github.com/brave/brave-ios/issues/7175))
+- Fixed reporting Brave wallet usage stats ([#7143](https://github.com/brave/brave-ios/issues/7143))
+- Fixed issue adding a custom network via panel prompt ([#6965](https://github.com/brave/brave-ios/issues/6965))
+- Fixed issue with ERC721 NFTs with different token identifiers showing incorrect balance ([#6954](https://github.com/brave/brave-ios/issues/6954))
+- Added support for Solana Name Service (SNS) navigation in app ([#6706](https://github.com/brave/brave-ios/issues/6706))
```

Comment:
Should be listed before the fixed line items. General order for notes line item is 
* Added/Implemented 
* Removed/Deleted
* Updated
* Fixed

The above order should be followed for general ui items as well